tests/ios-detector-spec.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+import chai from 'chai';
+
+import {
+  applyDetection,
+  isIOSDevice
+} from '../src/ios-detector';
+
+import {
+  forEachUserAgent,
+  restoreNavigator,
+  setUserAgent,
+  userAgents
+} from './helpers/user-agents.js';
+
+const htmlEl = document.documentElement;
+const iosPlatforms = [
+  'iPad',
+  'iPhone',
+  'iPod'
+];
+
+chai.should();
```

Comment:
I am biased against `should` as it has [limitations](http://chaijs.com/guide/styles/) when it comes to certain types of tests. I'm aware that there are workarounds, but `expect` doesn't have those limitations.